packages/data-manipulation/html/src/escaping.ts

```
@@ -0,0 +1,87 @@
+ * @returns The escaped string.
+ * @example
+ * escapeHtml('<script>alert("xss")</script>');
+ * // => '&lt;script>alert("xss")&lt;/script>'
+ * @example
+ * escapeHtml('<div>', true);
+ * // => '&lt;div>'
+ * @example
+ * escapeHtml('value="test"', true);
+ * // => 'value=&quot;test&quot;'
+ */
+const escapeHtml = (value: unknown, isAttribute: boolean = false): string => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'escapeHtml'. Consider to delete it or rename it.

'escapeHtml' is defined here:


(lint/suspicious/noRedeclare)